src/components/steps/step_horizontal.styles.ts

```diff
@@ -28,6 +29,22 @@ export const euiStepHorizontalStyles = (euiThemeContext: UseEuiTheme) => {
    *    e.g. due to some of their titles wrapping to multiple lines
    */
 
+  const _generateStepSizeAndInset = (size: EuiStepsHorizontalSizes) => {
+    const stepNumberSize =
+      size === 's' ? euiStep.numberXSSize : euiStep.numberSize;
+
+    return css`
+      &::before,
+      &::after {
+        inline-size: calc(50% - (${stepNumberSize} / 2));
+        inset-block-start: ${mathWithUnits(
+          [euiTheme.size.l, stepNumberSize],
+          (x, y) => x + y / 2
+        )};
+      } /* 1 */
```

Comment:
```suggestion
      }
```
This comment does not apply to this CSS